bukkit-legacy/src/main/java/com/viaversion/viaversion/bukkit/util/LegacyBlockToItem.java

```diff
@@ -27,20 +27,15 @@
 import java.util.logging.Level;
 
 public class LegacyBlockToItem {
-    private static final LegacyBlockToItem INSTANCE;
+
     private static final Random RANDOM = new Random();
+    private static LegacyBlockToItem INSTANCE;
```

Comment:
I don't think there's a good reason for this to stop being final, it doesn't seem to be changed anywhere except where it's initialized

Reply:
I did that in my cleanup commit to remove the local field in the static block, but if the silent fail change has to reverted anyway, this can be final again